src/PaymentPart/Output/HtmlOutput/HtmlOutput.php

```diff
@@ -0,0 +1,182 @@
+        }
+
+        $paymentPart = str_replace('{{ information-content }}', $informationContent, $paymentPart);
+
+        return $paymentPart;
+    }
+
+    private function addInformationContentReceipt(string $paymentPart): string
+    {
+        $informationContent = '';
+
+        foreach($this->getInformationElementsOfReceipt() as $informationElement) {
+            $informationContent .= $this->getContentElement($informationElement);
+        }
+
+        $paymentPart = str_replace('{{ information-content-receipt }}', $informationContent, $paymentPart);
+
+        return $paymentPart;
+    }
+
+    private function addCurrencyContent(string $paymentPart): string
+    {
+        $currencyContent = '';
+
+        foreach($this->getCurrencyElements() as $currencyElement) {
+            $currencyContent .= $this->getContentElement($currencyElement);
+        }
+
+        $paymentPart = str_replace('{{ currency-content }}', $currencyContent, $paymentPart);
+
+        return $paymentPart;
+    }
+
+    private function addAmountContent(string $paymentPart): string
+    {
+        $amountContent = '';
+
+        foreach($this->getAmountElements() as $amountElement) {
+            $amountContent .= $this->getContentElement($amountElement);
+        }
+
+        $paymentPart = str_replace('{{ amount-content }}', $amountContent, $paymentPart);
+
+        return $paymentPart;
+    }
+
+    private function addAmountContentReceipt(string $paymentPart): string
+    {
+        $amountContent = '';
+
+        foreach($this->getAmountElementsReceipt() as $amountElement) {
+            $amountContent .= $this->getContentElement($amountElement);
+        }
+
+        $paymentPart = str_replace('{{ amount-content-receipt }}', $amountContent, $paymentPart);
+
+        return $paymentPart;
+    }
+
+    private function addFurtherInformationContent(string $paymentPart): string
+    {
+        $furtherInformationContent = '';
+
+        foreach($this->getFurtherInformationElements() as $furtherInformationElement) {
+            $furtherInformationContent .= $this->getContentElement($furtherInformationElement);
+        }
+
+        $paymentPart = str_replace('{{ further-information-content }}', $furtherInformationContent, $paymentPart);
+
+        return $paymentPart;
+    }
+
+    private function addPrintableContent(string $paymentPart): string
+    {
+        $printableStyles = '';
+        if ($this->isPrintable()) {
+            $printableStyles = PrintableStylesTemplate::TEMPLATE;
+        }
+
+        $paymentPart = str_replace('{{ printable-content }}', $printableStyles, $paymentPart);
+
+        return $paymentPart;
+    }
+
+    private function getContentElement(OutputElementInterface $element): string
+    {
+        if ($element instanceof Title) {
+            $elementTemplate = TitleElementTemplate::TEMPLATE;
+            $elementString = str_replace('{{ title }}', $element->getTitle(), $elementTemplate);
```

Comment:
Instead of getting static content and replacing strings, it could be a class with a method that for, for example, parameter "title" will return HTML content string.
It relates to all `Template/...` classes.